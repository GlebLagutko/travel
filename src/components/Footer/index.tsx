import React from 'react';

export default function Footer() {


    return (
        <footer>
            <a target="_blank" href="https://github.com/GlebLagutko">GitHub</a>
            <span>2021</span>
            <img width="80px" height="50px" src="https://rs.school/images/rs_school_js.svg" alt="Icon"/>
            <a target="_blank" href="https://rs.school/js/">RS School</a>
        </footer>
    );
}