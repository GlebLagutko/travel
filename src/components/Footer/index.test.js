import {create} from 'react-test-renderer';

import React from "react";
import Footer from "./index";

describe('Footer', () => {
  it('renders correctly', async () => {

    const testRenderer = create(
      <Footer/>
    );

    expect(testRenderer.toJSON()).toMatchSnapshot();

  });

});
