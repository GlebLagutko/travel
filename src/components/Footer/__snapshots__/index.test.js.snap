// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer>
  <a
    href="https://github.com/GlebLagutko"
    target="_blank"
  >
    GitHub
  </a>
  <span>
    2021
  </span>
  <img
    alt="Icon"
    height="50px"
    src="https://rs.school/images/rs_school_js.svg"
    width="80px"
  />
  <a
    href="https://rs.school/js/"
    target="_blank"
  >
    RS School
  </a>
</footer>
`;
